src/views/AnnualReport.vue

```diff
@@ -43,12 +43,15 @@
     />
 
     <!-- Initial Page Load Transition -->
-    <div class="loading-container auto-fade-out">
-      <div class="loading__content">
-        <v-progress-circular color="primary" :size="50" indeterminate></v-progress-circular>
-        <div class="loading-msg">{{loadingMessage}}</div>
+    <v-fade-transition>
+      <!-- <div class="loading-container" v-show="showLoadingContainer"> -->
```

Comment:
This is preparation for a future bug that I almost fixed in this PR...

The problem is that it's not just the parent component that awaits certain things (like fetching a draft) -- it's also the child component(s) (like OfficeAddresses and Directors) that await the fetching of their data from the API. ATM the parent doesn't know the child(ren) is(are) busy so I can't properly implement the "showLoadingContainer" variable.

It's the same problem in COA and COD filing pages.

Reply:
Fixed.